quinn/src/streams.rs

```diff
@@ -11,7 +11,7 @@ use futures::{
     io::{AsyncRead, AsyncWrite},
     ready, FutureExt,
 };
-use proto::{ConnectionError, FinishError, StreamId};
+use proto::{Chunk, ConnectionError, FinishError, StreamId};
```

Comment:
We need to also reexport this at the crate level.

Reply:
I think that's there already?

Reply:
Done.